test/e2e/resourceinterpreter_test.go

```diff
@@ -0,0 +1,112 @@
+		workloadNamespace := testNamespace
+		workloadName := policyName
+
+		workload := testhelper.NewWorkload(workloadNamespace, workloadName)
+		policy := testhelper.NewPropagationPolicy(policyNamespace, policyName, []policyv1alpha1.ResourceSelector{
+			{
+				APIVersion: workload.APIVersion,
+				Kind:       workload.Kind,
+				Name:       workload.Name,
+			},
+		}, policyv1alpha1.Placement{
+			ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			},
+		})
+
+		ginkgo.It("InterpretReplica testing", func() {
+			framework.CreatePropagationPolicy(karmadaClient, policy)
+			framework.CreateWorkload(dynamicClient, workload)
+
+			ginkgo.By("check if workload's replica is interpreted", func() {
+				resourceBindingName := names.GenerateBindingName(workload.Kind, workload.Name)
+				expectedReplicas := int32(*workload.Spec.Replicas)
+
+				gomega.Eventually(func(g gomega.Gomega) (int32, error) {
+					resourceBinding, err := karmadaClient.WorkV1alpha2().ResourceBindings(workload.Namespace).Get(context.TODO(), resourceBindingName, metav1.GetOptions{})
+					g.Expect(err).NotTo(gomega.HaveOccurred())
+
+					klog.Infof(fmt.Sprintf("ResourceBinding(%s/%s)'s replicas is %d, expected: %d.",
+						resourceBinding.Namespace, resourceBinding.Name, resourceBinding.Spec.Replicas, expectedReplicas))
+					return resourceBinding.Spec.Replicas, nil
+				}, pollTimeout, pollInterval).Should(gomega.Equal(expectedReplicas))
+			})
+
+			framework.RemoveWorkload(dynamicClient, workload.Namespace, workload.Name)
+			framework.RemovePropagationPolicy(karmadaClient, policy.Namespace, policy.Name)
+		})
+	})
+
+	ginkgo.Context("InterpreterOperation Retain testing", func() {
+		policyNamespace := testNamespace
+		policyName := workloadNamePrefix + rand.String(RandomStrLength)
+		workloadNamespace := testNamespace
+		workloadName := policyName
+
+		workload := testhelper.NewWorkload(workloadNamespace, workloadName)
+		policy := testhelper.NewPropagationPolicy(policyNamespace, policyName, []policyv1alpha1.ResourceSelector{
+			{
+				APIVersion: workload.APIVersion,
+				Kind:       workload.Kind,
+				Name:       workload.Name,
+			},
+		}, policyv1alpha1.Placement{
+			ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			},
+		})
+
+		ginkgo.It("Retain testing", func() {
+			framework.CreatePropagationPolicy(karmadaClient, policy)
+			framework.CreateWorkload(dynamicClient, workload)
+
+			ginkgo.By("update workload's spec.paused to true", func() {
+				for _, cluster := range framework.ClusterNames() {
+					clusterDynamicClient := framework.GetClusterDynamicClient(cluster)
+					gomega.Expect(clusterDynamicClient).ShouldNot(gomega.BeNil())
+
+					clusterWorkload := framework.GetWorkload(clusterDynamicClient, workloadNamespace, workloadName)
+					framework.UpdateWorkload(clusterDynamicClient, clusterWorkload, cluster)
```

Comment:
You haven't updated the workload's `spec.paused` here

Reply:
Oh, carelessly. Thanks.